monte_carlo.py

```diff
@@ -43,7 +43,7 @@
  @param     mesh a Mesh object containing information about the mesh
 '''
 def transport_neutron(materials, bounds, tallies, fission_banks, first_round,
-        mesh, batch):
+        mesh, batch, chi):
```

Comment:
``chi`` should be a material property and therefore be included in ``materials``